Nodejs/Setup/NodejsTools/NodejsToolsFiles.proj

```diff
@@ -17,6 +17,7 @@
         <File Include="Microsoft.NodejsTools.ProjectWizard.dll" />
         <File Include="Microsoft.NodejsTools.Analysis.dll" />
         <File Include="Microsoft.NodejsTools.TestAdapter.dll" />
+        <File Include="Microsoft.NodejsTools.Telemetry.dll" />
```

Comment:
The setup code itself should remain the same in both repos, though. Otherwise we have to deal with keeping the installers in sync.